mlflow/main.tf

```
@@ -0,0 +1,8 @@
+resource "juju_model" "kubeflow" {
+  count = var.create_model ? 1 : 0
+  name  = local.model_name
+}
+
+locals {
+  model_name = "kubeflow"
+}
```

I used `locals` in kubeflow since kubeflow cannot be deployed in a model of different name. I 'd suggest we use the input `model_name` in this case.
```suggestion
resource "juju_model" "kubeflow" {
  count = var.create_model ? 1 : 0
  name  = var.model_name
}
```
And we could have a default value in that variable